src/main/resources/openapi/usermanagement.yaml

```diff
@@ -197,6 +201,26 @@ paths:
                 type: AUTHORISATION_109
                 title: User is not authorised for this endpoint
                 status: 403
+        '404':
+          description: Not Found Error
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
+              example:
+                type: "USER_MANAGEMENT_104"
+                title: "The provided security group does not exist"
```

Comment:
Should the corresponding USER_MANAGEMENT_104 title here be `The provided courthouse does not exist`?